unions-intersections/Exercise1/solution/Solution.ts

```diff
@@ -0,0 +1,25 @@
+interface Dog {
+  animalType: "canine";
+  age: number;
+  name: string;
+}
+
+interface Cat {
+  animalType: "feline";
+  age: number;
+  name: string;
+}
+
+type Pet = Cat | Dog; // Fixed!
+
+const cat: Pet = {
+  animalType: "feline",
+  age: 10,
+  name: "Garfield",
+};
+
+const dog: Pet = {
```

Comment:
Same thing in this PR as the other one—I think we should add a `console.info` of these variables so the solution doesn’t end up with any extraneous warnings.